brainzutils/musicbrainz_db/release_group.py

```diff
@@ -163,3 +163,55 @@ def _get_release_groups_for_artist_query(db, artist_id, release_types):
         join(models.ArtistCreditName, models.ArtistCreditName.artist_credit_id == models.ReleaseGroup.artist_credit_id).\
         join(models.Artist, models.Artist.id == models.ArtistCreditName.artist_id).\
         filter(models.Artist.gid == artist_id).filter(models.ReleaseGroupPrimaryType.name.in_(release_types))
+
+
+def get_release_groups_for_label(label_id, release_types=None, limit=None, offset=None):
+    """Get all release groups linked to a label.
+
+    Args:
+        label_id (uuid): MBID of the label.
+        release_types (list): List of types of release groups to be fetched.
+        limit (int): Max number of release groups to return.
+        offset (int): Offset that can be used in conjunction with the limit.
+
+    Returns:
+        Tuple containing the list of dictionaries of release groups ordered by release year
+        and the total count of the release groups.
+    """
+    label_id = str(label_id)
+    includes_data = defaultdict(dict)
+    if release_types is None:
+        release_types = []
+    release_types = [release_type.lower() for release_type in release_types]
+    # map release types to their case sensitive name in musicbrainz.release_group_primary_type table in the database
+    release_types_mapping = {
+        'album': 'Album',
+        'single': 'Single',
+        'ep': 'EP',
+        'broadcast': 'Broadcast',
+        'other': 'Other'
+    }
+    release_types = [release_types_mapping[release_type] for release_type in release_types]
+    with mb_session() as db:
+        release_groups_query = _get_release_groups_for_label_query(db, label_id, release_types)
+        count = release_groups_query.count()
+        release_groups = release_groups_query.order_by(
+            case([(models.ReleaseGroupMeta.first_release_date_year.is_(None), 1)], else_=0),
```

Comment:
I've not used it before, but does the [`nullslast`](https://stackoverflow.com/questions/44375919/in-sqlalchemy-is-there-a-way-to-sort-so-that-empty-cells-are-at-the-end-irrespe) modifier work here?

Reply:
We can update this in the release_groups_for_artist method too

Reply:
Yes it works nicely here. I've updated it for both the methods.